### DISCUSSION TITLE
What is time relative

### DISCUSSION THREAD

**Discussion by dr-costas** [mlflow/mlflow#8417] [Q&A]
Hi, 

As the title says, what is "Time (relative)" in MLFlow plots? I get that is relative to the start of the run, but what are the units of the x axis? Are they seconds? milliseconds? Tens of seconds? What?

**Comment by rahatzamancse** [comment]
Still no answer for this :(

**Comment by B-Step62** [comment]
"Time (relative)" on the x-axis represents the elapsed time since the first metric was logged in a run, and the units are seconds.
